### PR TITLE
Main-copy

### PR DESCRIPTION
**Pull request by sinisantino**
Updates client and server communications scripts to working versions

### CODE REVIEW

File: Team-Communicate/data_collection_server.py
Comment (TaylorBrown96) [inline Team-Communicate/data_collection_server.py:1]: # Suggested edits
+ If you're going to delete a lot of comments your should replace them with a comment block describing what they are used for. 
+ You shouldn't have the user manually enter their directory information. you should have it setup to initialize based on os.getcwd() as the base path and propagate from there.
+ Same thing for the IP address you can run console commands within your python methods and have it saved to a variable. [Example](https://www.tutorialspoint.com/python-program-to-find-the-ip-address-of-the-client)
+ There needs to be more detail within the docstrings for the methods that tells the user what is getting passed, what is happening and what is getting returned.

File: Team-Communicate/data_collection_client.py
Comment (TaylorBrown96) [inline Team-Communicate/data_collection_client.py:1]: ### Questions
For line 62 & 66 why did you change the message prompts to something ambiguous?
**AND**
Why did you remove the timestamp feature from the file naming convention?

**Comment by TaylorBrown96** [comment]
## Suggested edits
+ If you're going to delete a lot of comments your should replace them with a comment block describing what they are used for. 
+ You shouldn't have the user manually enter their directory information. you should have it setup to initialize based on os.getcwd() as the base path and propagate from there.
+ Same thing for the IP address you can run console commands within your python methods and have it saved to a variable. [Example](https://www.tutorialspoint.com/python-program-to-find-the-ip-address-of-the-client)
+ There needs to be more detail within the docstrings for the methods that tells the user what is getting passed, what is happening and what is getting returned.

### Questions
For line 62 & 66 why did you change the message prompts to something ambiguous?
**AND**
Why did you remove the timestamp feature from the file naming convention?

**Comment by sinisantino** [comment]
@TaylorBrown96 

**As a preface, the reason I requested this was because the current script wasn't functional. The timestamp and directory aspects of the script weren't functional so I removed them in order to test them on their own branch before merging them.**

1) **If you're going to delete a lot of comments your should replace them with a comment block describing what they are used for.** I did this mostly because I thought the comments were redundant. It seemed like they were explaining the packages in a superficial/general way. I usually don't put comments for the packages. 

2) **You shouldn't have the user manually enter their directory information. you should have it setup to initialize based on os.getcwd() as the base path and propagate from there. Same thing for the IP address you can run console commands within your python methods and have it saved to a variable. [Example](https://www.tutorialspoint.com/python-program-to-find-the-ip-address-of-the-client) You shouldn't have the user manually enter their directory information. you should have it setup to initialize based on os.getcwd() as the base path and propagate from there.** I wasn't familiar with that method at all. I can change that for sure in future versions. 

3) **There needs to be more detail within the docstrings for the methods that tells the user what is getting passed, what is happening and what is getting returned.** That makes sense, I can clarify it on the future version. 

4) **For line 62 & 66 why did you change the message prompts to something ambiguous?** This version with the ambiguous messages was the original version that I received from Matt and Garrett. I changed them to the more specific the last edit that I did. However, I forgot that I changed them to more specific messages because I was reverting it to the original from M and G. 

5) **Why did you remove the timestamp feature from the file naming convention?** I removed this because it didn't work. I am still getting familiar with github. I added the timestamp without testing it in person. I realized yesterday that it didn't work, so I removed it to at least have a code that functioned like the original without any modifications. I created a new branch titled Santino_S_timestamp where I can test the timestamp and directory functions prior to pushing them to the main branch.

**Review by TaylorBrown96 — CHANGES_REQUESTED** [review]
Please add in the missing information (codeblocks and docstring) and make the necessary changes covered by "in the next version" prior to trying to push to the main branch. **Refer back to the comment left on the pull request "Conversation" tab for guidance**

The main branch is for completed sections of code that have been thoroughly verified and vilified.
